hashes/src/cmp.rs

```diff
@@ -15,6 +15,13 @@
 /// As of rust 1.31.0 disassembly looks completely within reason for this, see
 /// <https://godbolt.org/z/mMbGQv>.
 pub fn fixed_time_eq(a: &[u8], b: &[u8]) -> bool {
+    #[cfg(hashes_fuzz)]
+    {
+        // Fuzzers want to break memcmp calls into separate comparisons for coverage monitoring,
+        // so we avoid our fancy fixed-time comparison below for fuzzers.
+        return a == b;
+    }
```

Comment:
Concept ACK but this needs either a `#[cfg(not(hashes_fuzz))]` (preferable, I think) or `#[allow(unreachable)]`